toonz/sources/translations/german/tnztools.ts

```diff
@@ -0,0 +1,1005 @@
+    </message>
+    <message>
+        <source>Distance:</source>
+        <translation>Abstand：</translation>
+    </message>
+    <message>
+        <source>Style Index:</source>
+        <translation>Stil-Index:</translation>
+    </message>
+    <message>
+        <source>Opacity:</source>
+        <translation>Opazität:</translation>
+    </message>
+    <message>
+        <source>Frame Range</source>
+        <translation>Frame-Reichweite</translation>
+    </message>
+    <message>
+        <source>Angle:</source>
+        <translation>Winkel：</translation>
+    </message>
+</context>
+<context>
+    <name>SelectionTool</name>
+    <message>
+        <source>Type:</source>
+        <translation>Typ:</translation>
+    </message>
+</context>
+<context>
+    <name>SelectionToolOptionsBox</name>
+    <message>
+        <source>H:</source>
+        <translation>H:</translation>
+    </message>
+    <message>
+        <source>V:</source>
+        <translation>V:</translation>
+    </message>
+    <message>
+        <source>Rotation</source>
+        <translation>Rotation</translation>
+    </message>
+    <message>
+        <source>E/W:</source>
+        <translation>X:</translation>
+    </message>
+    <message>
+        <source>N/S:</source>
+        <translation>Y:</translation>
+    </message>
+    <message>
+        <source>Thickness</source>
+        <translation>Dicke</translation>
+    </message>
+    <message>
+        <source>Link</source>
+        <translation>Seitenverhältnis festlegen</translation>
+    </message>
+</context>
+<context>
+    <name>SkeletonTool</name>
+    <message>
+        <source>Global Key</source>
+        <translation>Globale Taste</translation>
+    </message>
+    <message>
+        <source>Show Only Active Skeleton</source>
+        <translation>Nur aktives Skelett anzeigen</translation>
+    </message>
+    <message>
+        <source>Mode:</source>
+        <translation>Modus:</translation>
+    </message>
+    <message>
+        <source>Reset Pinned Center</source>
+        <translation>Festgesetzte Mitte zurücksetzen</translation>
+    </message>
+</context>
+<context>
+    <name>TrackerTool</name>
+    <message>
+        <source>Width:</source>
+        <translation>Breite:</translation>
+    </message>
+    <message>
+        <source>Height:</source>
+        <translation>Höhe:</translation>
+    </message>
+    <message>
+        <source>X:</source>
+        <translation>X:</translation>
+    </message>
+    <message>
+        <source>Y:</source>
+        <translation>Y:</translation>
+    </message>
+</context>
+<context>
+    <name>TypeTool</name>
+    <message>
+        <source>Font:</source>
+        <translation>Schriftsatz:</translation>
+    </message>
+    <message>
+        <source>Style:</source>
+        <translation>Stil:</translation>
+    </message>
+    <message>
+        <source>Vertical Orientation</source>
+        <translation>Vertikale Orientierung</translation>
```

Comment:
Consider using the German term Ausrichtung instead of the foreign word Orientierung.